cypress/e2e/example.cy.js

```diff
@@ -1,51 +1,51 @@
-import { faker } from '@faker-js/faker';
-/// <reference types='cypress' />
-
-describe('Bank app', () => {
-  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
-  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
-  const balance = depositAmount - withdrawAmount;
-  const user = 'Harry Potter';
-  const accountNumber = '1004';
-
-  before(() => {
-    cy.visit('/');
-  });
-
-  it('should provide the ability to work with bank account', () => {
-    cy.contains('.btn', 'Customer Login').click();
-    cy.get('[name="userSelect"]').select(user);
-    cy.contains('.btn', 'Login').click();
-
-    cy.contains('[ng-hide="noAccount"]', 'Account Number')
-      .contains('strong', accountNumber)
-      .should('be.visible');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', '0')
-      .should('be.visible');
-    cy.contains('.ng-binding', 'Dollar')
-      .should('be.visible');
-
-    cy.get('[ng-click="deposit()"]').click();
-    cy.get('[placeholder="amount"]').type(depositAmount);
-    cy.contains('[type="submit"]', 'Deposit').click();
-
-    cy.get('[ng-show="message"]')
-      .should('contain', 'Deposit Successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', depositAmount)
-      .should('be.visible');
-
-    cy.get('[ng-click="withdrawl()"]').click();
-    cy.contains('[type="submit"]', 'Withdraw')
-      .should('be.visible');
-    cy.get('[placeholder="amount"]').type(withdrawAmount);
-    cy.contains('[type="submit"]', 'Withdraw').click();
-
-    cy.get('[ng-show="message"]')
-      .should('contain', 'Transaction successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', balance)
-      .should('be.visible');
-  });
-});
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  const balance = depositAmount - withdrawAmount;

```

Comment:
The `depositAmount` and `withdrawAmount` are strings due to template literals. This causes `balance` to be a string as well, leading to incorrect arithmetic operations. Consider converting them to numbers before performing arithmetic operations.